dist/windows/config.nsh

```diff
@@ -25,20 +14,20 @@
 ; 4.5.1.3 -> good
 ; 4.5.1.3.2 -> bad
 ; 4.5.0beta -> bad
-!define /ifndef QBT_VERSION "4.5.0"
+!define /ifndef QBT_VERSION "5.0"
 
 ; Option that controls the installer's window name
 ; If set, its value will be used like this:
 ; "qBittorrent ${QBT_INSTALLER_FILENAME}"
-; If not set, the window name will be auto composed from QBT_VERSION, QBT_USES_QT6, QBT_IS_X64
+; If not set, the window name will be auto composed from QBT_VERSION
 ; If you set this define then you MUST set QBT_INSTALLER_FILENAME too. Otherwise it will be ignored.
 ; This define is meant to ease automation from scripts/commandline
 ;!define QBT_INSTALLER_WINDOWNAME
 
 ; Option that controls the installer's window name
 ; If set, its value will be used like this:
 ; "qbittorrent_${QBT_INSTALLER_FILENAME}_setup.exe"
-; If not set, the window name will be auto composed from QBT_VERSION, QBT_USES_QT6, QBT_IS_X64
+; If not set, the window name will be auto composed from QBT_VERSION
```

Comment:
"the window name" -> "the installer filename"

Reply:
> "the window name" -> "the installer filename"

Done.